plugin/src/test/java/org/opensearch/ml/rest/MLCommonsRestTestCase.java

```diff
@@ -276,7 +276,7 @@ protected void wipeAllODFEIndices() throws IOException {
 
             for (Map<String, Object> index : parserList) {
                 String indexName = (String) index.get("index");
-                if (indexName != null && !".opendistro_security".equals(indexName)) {
+                if (indexName != null && !".opendistro_security".equals(indexName) && !".plugins-ml-config".equals(indexName)) {
```

Comment:
I don't think we can just do this here. Because for single tenancy we are testing if the index is being created through the code or not. May be only for multi-tenancy we can do this.

Reply:
Yes I know.  

Problem is it’s annotated @after so can’t be overridden.  Need a better long term fix. 

Possibly delegate to another overridable method.

Reply:
Actually, this fix probably isn't needed, it's just useful to isolate the problem.  This is actually the local index involved here.

What we have is a race condition:
1. When the first test starts it creates a (local) ML Config index (which isn't needed)
2. When the first test finishes it issues a rest call in the `@After` of the superclass to delete the (local) index. This is answered by the primary shard (giving a successful request) but the cluster state isn't instantly updated.
3. The second test starts and checks the cluster state and thinks the index is there, but then it has to check the index mapping and by that time it has been deleted and results in the error.

The error probably exists in all the other ML Commons integ tests, but the unique part about our tests is that several start out immediately creating a connector and calling `encrypt()`.  Let me try throwing a refresh call at the start of every test method to ensure cluster state is caught up and see if it fixes....

Reply:
And refreshing worked.  Going to file this as a bug on `main`.